QuickDeployment/QuickDeploymentSslSetup.md

```diff
@@ -1,17 +1,21 @@
 # Quick Deployment Environment SSL Setup
 
-All services have been protected with Self-Signed SSL certificates and are placed in the appropriate stores. Under the following situations you would want to run the script that follows:
-* If you want to expose this to the internet so clients can connect from outside your network
-* If you change the hostname of this server
-* If you add the QDE to a domain
-* If you would like to use your own SSL/TLS certificates
+All services have been protected with Self-Signed SSL certificates and are placed in the appropriate stores.
+Under the following situations you would want to run the script that follows:
 
-```powershell
-Set-ExecutionPolicy Bypass -Scope Process -Force ; . C:\choco-setup\files\New-SslCertificates.ps1
-```
+- If you want to expose this to the internet so clients can connect from outside your network.
+- If you change the hostname of this server.
+- If you add the QDE to a domain.
+- If you would like to use your own SSL/TLS certificates.
 
-> :warning: **WARNING**: This script will seemingly prompt for input, and have other strange output. This is due to poor Java tooling and console output which cannot be suppressed. Just let things happen, as things are working as expected.
+> :warning: **WARNING**: This script will seemingly prompt for input, and have other strange output.
+> This is due to poor Java tooling and console output which cannot be suppressed.
+> Just let things happen, as things are working as expected.
```

Comment:
Eventually it would be good to capture a picture with this

Reply:
@steviecoaster do you happen to have a screenshot of this behaviour?

Reply:
Eventually,  I won't hold things up on this PR for it.